app/src/main/res/values/strings.xml

```diff
@@ -1,12 +1,13 @@
 <resources>
-    <string name="app_name">Demo App</string>
-    <string name="welcome_to_dimelo">Dimelo Mobile demo App</string>
-    <string name="how_to">Tap "Open Full Screen" to see the default Dimelo Mobile integration.</string>
-    <string name="telecom_welcome">Upgrade your plan, subscribe to services and contact us with just a tap.</string>
-    <string name="bank_welcome">Consult your accounts, transfer money and get in touch with just a tap.</string>
-    <string name="enable_thread">Enable Threads:</string>
-    <string name="user_identifier_text">Choose a user identifier:</string>
-    <string name="update">Update</string>
+    <string name="app_name" translatable="false">Demo App</string>
```

Comment:
is there any reason you added this `translatable="false"` everywhere?

Reply:
no just is the name of the application, you don't need to translate

Reply:
I mean you added `translatable="false"` in every string in this file, what is the reason for this change?

Reply:
translatable="false" we don't need to translate all this string even if we don't have the file of traduction